src/core/client/admin/components/BanModal.css

```diff
@@ -12,6 +12,31 @@ $ban-modal-text: var(--palette-text-500);
   height: calc(12 * var(--mini-unit));
 }
 
-.sitesToggle {
+.domainBanHeader {
+  margin-bottom: var(--spacing-2);
+
```

Comment:
lol, looks like we need to take a peek at our CSS auto-formatter at some point, it's letting us leave errant spaces behind